packages/portfolio-deploy/test/portfolio-start.core.test.ts

```diff
@@ -97,9 +102,9 @@ test('coreEval code without swingset', async t => {
   );
 
   const options = toExternalConfig(
-    harden({ axelarConfig, assetInfo: [], chainInfo: {}, net: 'local' }),
+    harden({ axelarConfig } as PortfolioDeployConfig),
```

Comment:
Consider adding a little comment that justifies the cast.

Reply:
yeah; as I mentioned, I'm starting to lean toward explicit justification for casts.

But now I see this is test code.